Project.toml

```diff
@@ -11,10 +11,12 @@ DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 DelimitedFiles = "8bb1440f-4735-579b-a4ab-409b98df4dab"
 DiffEqFlux = "aae7a2af-3d4f-5e19-a356-7da93b79d9d0"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+FFMPEG = "c87230d0-a227-11e9-1b43-d7ebe4e7570a"
```

Comment:
You don't want to add this here, it should just be transient through Plots, and it's a